.circleci/config.yml

```diff
@@ -101,5 +101,14 @@ jobs:
       - checkout
       - setup_remote_docker:
           docker_layer_caching: true
-      - run: make release-publish
-
+      - run:
+          name: 'Set custom environment variables'
+          command: |
+            echo 'export GIT_TAG=$(git describe --tags | tr -d "v")' >> $BASH_ENV
+            source $BASH_ENV
```

Comment:
Don't we get the latest git tag from Circle already? `CIRCLE_TAG` We could short-cut this to aliasing that env var.

https://circleci.com/docs/2.0/env-vars/#built-in-environment-variables

Reply:
I was having issues with passing the built-in environment variables to the bash script. Then I found some article/post that CircleCI won't pass the built-in variables to scripts for security reasons, so this is their prescribed workaround. Now that I have things stable, I can give the built-in env vars another try to see if I just missed something earlier on.

Reply:
Closing the loop. Here's the fine print regarding what I mentioned for the built-in env vars, which is part of the docs section linked above.

> Note: You cannot use a built-in environment variable to define another environment variable. Instead, you must use a `run` step to export the new environment variables using `BASH_ENV`.

Seems a little odd, but at least what we've currently got works.